### DISCUSSION TITLE
Access  Code

### DISCUSSION THREAD

**Discussion by chainmeme** [tanshuai/alphabiz#24] [General]
Hello

How to get a access to Mac download? I am interested in testing this software.

Thank you

**Comment by tanshuai** [comment]
Thank you for interested in our software.

Please find the latest release of MacOS (.dmg) from https://github.com/tanshuai/alphabiz/releases

**Comment by KRUELCS** [comment]
Hello

How can I run this on codespaces?

Thank You
